pep_sphinx_extensions/pep_zero_generator/parser.py

```diff
@@ -127,6 +127,25 @@ def details(self, *, title_length) -> dict[str, str | int]:
             "authors": ", ".join(author.nick for author in self.authors),
         }
 
+    @property
+    def full_details(self) -> dict[str, str]:
+        """Returns all headers of the PEP as a dict."""
```

Comment:
```suggestion
        """Return all headers of the PEP as a dict."""
```

Correct imperative tense, per PEP 257 (PEP-0257) and standard docstring convention.